packages/calypso-e2e/src/lib/pages/plugins-page.ts

```diff
@@ -17,8 +17,7 @@ const selectors = {
 	pluginTitleOnSection: ( section: string, plugin: string ) =>
 		`.plugins-browser-list:has(.plugins-browser-list__title.${ section }) :text-is("${ plugin }")`,
 	sectionTitles: '.plugins-browser-list__title',
-	browseAllFree: 'a[href^="/plugins/browse/popular"]',
-	browseAllPaid: 'a[href^="/plugins/browse/paid"]',
+	browseAllBtn: ( section: string ) => `text=${ section }Browse All >> a`,
```

Comment:
**suggestion::blocking**: Btn is non-standard terminology in our test suite framework. Please use the term `Button`.

Reply:
Good to know @worldomonation. Thanks!